assets/src/settings-page/settings-footer.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { useContext, useEffect } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Options } from '../components/options-context-provider';
+import { ReaderThemes } from '../components/reader-themes-context-provider';
+
+/**
+ * The bottom section of the settings page.
+ */
+export function SettingsFooter() {
+	const { didSaveOptions, editedOptions, saveOptions, savingOptions } = useContext( Options );
+	const { downloadingTheme } = useContext( ReaderThemes );
+
+	const { reader_theme: readerTheme, theme_support: themeSupport } = editedOptions;
+
+	/**
+	 * Submits the PHP-generated form on the page after options have saved via REST.
+	 */
+	useEffect( () => {
+		if ( true === didSaveOptions && ! downloadingTheme ) {
+			document.querySelector( 'form#amp-settings' ).submit();
+		}
+	}, [ didSaveOptions, downloadingTheme ] );
+
+	const disabled = ! themeSupport ||
+		savingOptions ||
+		didSaveOptions ||
+		downloadingTheme ||
+		( 'reader' === themeSupport && ! readerTheme );
+
+	return (
+		<section className="settings-footer">
+			<Button isPrimary onClick={ saveOptions } disabled={ disabled }>
+				{ __( 'Save changes', 'amp' ) }
```

Comment:
Once we use the REST API exclusively for updating changes, there will need to be a success message when saved.

Reply:
Yup, when we don't have backend form handling we'll have a success notice of some kind.